src/components/Game.js

```diff
@@ -33,13 +34,19 @@ function Game() {
     /* eslint-disable-next-line */
       alert('Game over!'); 
       setIsCounting(false);
+      setRoundOverStage(true);
       setAvailableWords(words);
       setTimeLeft(1 * 60);
     }
   };
 
   const handleStart = () => {
     setIsCounting(true);
+    setCorrectScore(0);
+    nextWord();
+  };
+
+  const handleNextWord = () => {
```

Comment:
no need to create a separate callback for this, u can assign this directly on string 80

```
<MyButton onClick={nextWord}>Next</MyButton>
```